core/src/main/java/io/spine/core/Events.java

```diff
@@ -93,10 +93,11 @@ public static EventMessage ensureMessage(Message eventOrMessage) {
      *         an ID to check
      * @throws IllegalArgumentException
      *         if the ID string value is empty or blank
+     * @deprecated please use {@link io.spine.util.Preconditions2#checkNotDefaultArg(Message)}
      */
+    @Deprecated
     public static EventId checkValid(EventId id) {
-        checkNotNull(id);
-        checkNotEmptyOrBlank(id.getValue(), "event ID");
+        checkNotDefaultArg(id);
         return id;
```

Comment:
I believe we can just: `return checkNotDefaultArg(id);` here. But while it's deprecated already, not sure if it matters.